### ISSUE TITLE
Expand the domain of medicalSpecialty to MedicalOrganization

### ISSUE THREAD

**Issue by jvandriel** [twamarc/ScheMed#33]
I suggest we expand the domain of `medicalSpecialty` to `schema.org/MedicalOrganization`.

Currently its domain is limited to `schema.org/Hospital`, `schema.org/MedicalClinic`and `schema.org/Physician` yet it would be very usefull to be able to specify it for other types of medical organizations as well (eg, a `schema.org/DiagnosticLab` that specializes in toxicology (`relevantSpecialty > schema.org/Toxicologic`), or a pediatric `schema.org/Dentist` (`relevantSpecialty > schema.org/Pediatric`)).

related to #14

**Comment by twamarc** [comment]
+1 it makes sense.

**Comment by LeezaRodriguez** [comment]
I am going to chime in here because I think that 'Medical Specialty' is the one opportunity we have to ~weed out the marketing scam that goes on at medical websites. If we tie it to all organizations it has the potential to get exploited. And once knowledge panels pick up the markup, it would  give the same credibility to quasi medical entities as legit entities.

A Medical Specialty is a very formal declaration of one's medical training credentials.   As a consumer, I expect that when I go to a 'Medical Specialty X' Clinic, that the doctors who run the clinic have formal residency training in the Medical Specialty X.  But that is not always the case.

I have always thought that 'Medical Specialty' has ~ provenance rooted in the physician.  I say this because it is the physician who gets a certificate after completing a formal residency (from an accredited program)  in a designated Medical Specialty.  He must then take oral and written exams, known as Board Certification. 

Any Healthcare organization can create an enterprise based on the Medical Specialty of the physicians it employs.  Reputable organizations hire physicians who are Board Certified in a designated Medical Specialty.  These physicians then oversee healthcare staff and technicians of the enterprise.

Similarly speaking,  do we allow an organization to mark up their website content in a 'Medical Specialty' for which the organization has NO formally trained physicians in that Medical Specialty? A good example of this is an organization who markets themselves as 'Pain Medicine', yet not a single one of their credentialed staff has completed a formal residency program in 'Physical Rehabilitation and Pain Management' . In this example, the people who run these pain clinics may be  M.D.s , but often times they have no residency training whatsoever in this 'Medical Specialty'.   The consumer loses.  All you have to do is watch 60 minutes to see how pain clinics are running rogue across America. And this is only one example in one medical specialty.  

I don't have a solution at the moment, but it would be great to take this into consideration in the description of the vocabulary if we apply this to organizations.  Here is the ABMS (American Board of Medical Specialties)  list of Medical Specialties and Subspecialty certificates:
http://www.abms.org/member-boards/specialty-subspecialty-certificates/

**Comment by jvandriel** [comment]
I understand your point @LeezaRodriguez, but don't you think scam be separated by filtering by unique identifiers like `naics` and `duns`?

And for your own site `sameAs` references to npiregistry have been specified, all means by which it should be possible separate the real deal form the scam IMHO.

**Comment by LeezaRodriguez** [comment]
In medicine, the NPI number carries more significance than the DUNS. Note that in the USA,  NPI numbers are given to individual health care providers AND organizations. DUNS is only for organizations and predominantly reflects the corporate and financial filings.   

NPI numbers for individual providers do list the official certificate 'Medical Specialty' of the provider , but I have to revisit the NPI organization choices.

If it is tied to organizations, perhaps something in the description which states that a member(s) of the organization must be credentialed or Board Certified in that Medical Specialty?  There just needs to be some technical threshold for a webmaster to designate the 'Medical Specialty' markup to an organization.

**Comment by jvandriel** [comment]
Might it be an idea than to add an `npi` property to `MedicalOrganization`? (new issue)

Any idea if there's an European equivalent for NPI @twamarc?

**Comment by LeezaRodriguez** [comment]
Good idea for USA.  Without an NPI number, one doesn't really exist as a medical entity (as an individual provider or as a provider organization).  If somebody knows otherwise, please speak up.

**Comment by LeezaRodriguez** [comment]
In the NPI database, here is an example search of Medical Provider Organizations in zip code 21093.  The list shows how it categorizes the top level taxonomy for these organizations. Only some organizations have an associated 'Medical Specialty'.  Click the NPI number to see the specifics for any organization.  

https://nppes.cms.hhs.gov/NPPESRegistry/NPIRegistryPaginate.do?lastName=&zip=21093&nextPage=1&searchType=org&city=&orgName=&state=&firstName=&searchNpi=&orgDba=

**Comment by twamarc** [comment]
@LeezaRodriguez @jvandriel : Nice discussions here. We should se how we finetune definitions of our terms to reflect this. Need a follow up. Can you take this discussion and suggest a final conclusion in coming CG meeting @jvandriel ? (it will be the last Friday of october).

@jvandriel : 
+1 Might it be an idea than to add an npi property to MedicalOrganization (new issue)

Q:Any idea if there's an European equivalent for NPI? No in Europe we have no unique common Healthcare Provider's Identifier for all countries.
Each country has it's own (in Belgium is INAMI/RIZIV, in France it's even complicated with multiple codes like CPS, tec.)
At European level there's an effort to harmonize those IDs by creating the " Doctors’ Professional Card" but I have no idea how far they are.
In conclusion when suggesting the new predicate as `npi` we have to suggest at the same time another more generic for other area, something like professional Ientifier number `pin`.
But here we have to take into account other existing properties in schema core for identifiers like `membershipNumber` etc.

My 2 cents

**Comment by twamarc** [comment]
seems to reach conclusion here about expanding the domain of `medicalSpecialty` to `schema.org/MedicalOrganization`.
need to open other issues Other related suggestions.
Followed in : https://github.com/schemaorg/schemaorg/issues/492

**Comment by LeezaRodriguez** [comment]
@twmarc  Hi Marc, I see that this is now closed. Is it possible to clarify the definition of MedicalSpecialty so that is reflects the formal process of training, certification, and licensing?  

I'm afraid that webmasters may apply the term loosely, without knowledge that the designation of a 'MedicalSpecialty' is quite a formal thing.  Any person or organization which is tied to a 'MedicalSpecialty' should have formal 'certification'  and be 'licensed'  in the Medical Specialty. The person or organization with a MedicalSpecialty must have attained **training, certification, and licensing from an authority organization to practice medicine in that Medical Specialty**.

**Current description:** Any specific branch of medical science or practice. Medical specialities include clinical specialties that pertain to particular organ systems and their respective disease states, as well as allied health specialties. Enumerated type.

**Proposed description:** Any specific branch of medical science or practice that a person or organization has formal training, certification and licensing from an authority organization to practice medicine in that Medical Specialty. Medical specialities include clinical specialties that pertain to particular organ systems and their respective disease states, as well as allied health specialties. Enumerated type.

Or something similar to speak to my concerns.

**Comment by twamarc** [comment]
@LeezaRodriguez I would rather open another issue about extending the definition. The change in the definition proposed here adds an important dimension about the quality of the person in terms of having training. I can imagine that some people are called specialists (without having trained in school) and some specialities has no formal training. So it's not straightforward, we need to collect other's view on this. That's why I support opening another issue for that.
BTW we can also discuss this in next week's TCon if you will attend

**Comment by LeezaRodriguez** [comment]
@twamarc  Great. When a physician asserts himself as a provider of care in a designated Medical Specialty, the consumer (and the AI engine) should be able to parse  **if that provider is trained/certified/boarded** or not.  HUGE difference. As the enumerated list of Medical Specialties grows, this really concerns me. 

How do we/I open this issue?

**Comment by twamarc** [comment]
@LeezaRodriguez I opened an issue for that: see https://github.com/twamarc/ScheMed/issues/34
